### PR TITLE
DAT-4469 | use proper variable + reduce log level error --> warning

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/DAT-4469

//cc @Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -164,8 +164,9 @@ public function getArticleComments() {
 	private function prepareWikiVariables() {
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 		$navigation = $this->getNavigation();
-		if ( empty( $navData ) ) {
-			\Wikia\Logger\WikiaLogger::instance()->error(
+
+		if ( empty( $navigation ) ) {
+			\Wikia\Logger\WikiaLogger::instance()->warning(
```
Comment (rogatty) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: See https://wikia-inc.atlassian.net/wiki/display/PLAT/Logging#Logging-severitylevelsMeaningfulseveritylevels

IMO this should be `debug` level as there is no action we can take here.

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 2]
```diff
@@ -164,8 +164,9 @@ public function getArticleComments() {
 	private function prepareWikiVariables() {
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 		$navigation = $this->getNavigation();
-		if ( empty( $navData ) ) {
-			\Wikia\Logger\WikiaLogger::instance()->error(
+
+		if ( empty( $navigation ) ) {
+			\Wikia\Logger\WikiaLogger::instance()->debug(
 				'Fallback to empty navigation'
```
Comment (bognix) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:170]: make it notice please. It was logged as an error before our code refactor back in January. On the other hand I am wondering if we ever going to do something with this information.

**Comment by bognix** [comment]
general wonderment on my side, beside LGTM

**Pull request merged**